docs/release/fundamentals/queries-and-expressions.ipynb

```diff
@@ -0,0 +1,2023 @@
+       "            </div></td>\n",
+       "      <td>None</td>\n",
+       "      <td>ValueError</td>\n",
+       "      <td>band index out of range</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td><div class=\"pxt_image\" style=\"width:28px;\">\n",
+       "                <img src=\"data:image/webp;base64,UklGRsQAAABXRUJQVlA4ILgAAADwBACdASocABwAPmEojkWkIqEb/VQAQAYEtIACIJfpf53anfwh5bFxZyxmg4dIzIAA/v8aTg5Pf4AaLvDOb2+nBLe7h4WLP4eah5+VtiBuH5fnAK/9o/+/4nl+i/XuYMSDuGzWvPP7PeYAYqDelo/gGeaJWXI9qBrwbupRut1RmHAmjHeJdu952t5DneGb8d1NHD67m1X/4Fr/ktLUCjUDM3A9y/Tmv/u48GyVebp9QUFiS+qkAAAA\" width=\"28\" />\n",
+       "            </div></td>\n",
+       "      <td>None</td>\n",
+       "      <td>ValueError</td>\n",
+       "      <td>band index out of range</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td><div class=\"pxt_image\" style=\"width:28px;\">\n",
+       "                <img src=\"data:image/webp;base64,UklGRhIBAABXRUJQVlA4IAYBAAAQBwCdASocABwAPmkqkEWkIqGYBgBABoS0gAiDrbQKlm9OvPM8p+ijvgH6ADs18j7y7UsovRG992j1lWJXI+NjsAD+/p9i31NdsRkVHtTBTbgEfcsiV5cS5IfF+yS7MDdvlaSHRre6nuYBWIGIMPNc7//PKE5FDkecaPH1C5KuSA/FmB0Mw15jQ4M0b5oct9Sv71vHA8P6IA5GrLvUA+xjJCnISNNaYSiHVONthArmojGAh1x2X9w6ER0Oe1Yqh//8NUVB//4iTpFaItcumEp4Ryr+jO63PtCFdy7qk9dlTU9Mno2GIXYCVLSAkv7l2L/rqd15c4R/MS1/O/u2yTOuN7uvmCQA\" width=\"28\" />\n",
+       "            </div></td>\n",
+       "      <td>None</td>\n",
+       "      <td>ValueError</td>\n",
+       "      <td>band index out of range</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>"
+      ],
+      "text/plain": [
+       "                                               image channel  \\\n",
+       "0  <PIL.JpegImagePlugin.JpegImageFile image mode=...    None   \n",
+       "1  <PIL.JpegImagePlugin.JpegImageFile image mode=...    None   \n",
+       "2  <PIL.JpegImagePlugin.JpegImageFile image mode=...    None   \n",
+       "3  <PIL.JpegImagePlugin.JpegImageFile image mode=...    None   \n",
+       "4  <PIL.JpegImagePlugin.JpegImageFile image mode=...    None   \n",
+       "\n",
+       "  channel_errortype         channel_errormsg  \n",
+       "0        ValueError  band index out of range  \n",
+       "1        ValueError  band index out of range  \n",
+       "2        ValueError  band index out of range  \n",
+       "3        ValueError  band index out of range  \n",
+       "4        ValueError  band index out of range  "
+      ]
+     },
+     "execution_count": 27,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "t.select(t.image, t.channel, t.channel.errortype, t.channel.errormsg).head(5)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6847abcb-1b78-4031-b097-8f4b7263b192",
+   "metadata": {},
+   "source": [
+    "## The Pixeltable Type System"
```

Comment:
I think this needs to be in the first of the fundamentals tutorials. That would be the very first question on my mind if I learn about a new database system: how do I define tables and what are the supported types.